py3dtiles/gltf_material.py

```diff
@@ -0,0 +1,71 @@
+class GlTFMaterial():
+    def __init__(self, metallicFactor=0, roughnessFactor=0, rgb=[1, 1, 1], alpha=1):
+        self.metallicFactor = metallicFactor
+        self.roughnessFactor = roughnessFactor
+        self.rgba = rgb
+        if len(rgb) < 4:
```

Comment:
You need to make sure that rgb have a length of 3.
Also, it may be possible to include the alpha value in the rgb directly (RGBA)

Reply:
@clementcolin Your comment looks like a change request. Please do toggle that flag in your review is this were to be the case.

Reply:
The color will always be written as RGBA in the GlTF header. Splitting the color and the alpha in the constructor seems more convenient (to me). Note that the alpha can be added directly in the `rgb` param of the constructor. In this case, the `alpha` param will be ignored.

Also, the `rgba.setter` ensures that its color has either 3 or 4 values. If it has only 3 values, it means that the 4th value will be added by the `alpha.setter`.